index.html

```diff
@@ -973,9 +973,9 @@
     </LI>
   <LI><FONT COLOR="#006633"><I>Instructions:</I></FONT> <B>1.</B>
     Extract all files, which will be contained
-    in a directory called "metamath". <B>2.</B> For Windows, double-click on
+    in a directory called "metamath". On Linux and Mac OS this can be done using the popular "tar&nbsp;xvzf <your_download_of_metamath.tar.gz>" command.  <B>2.</B> For Windows, double-click on
```

Comment:
For angle brackets, that should be `&lt;` and `&gt;` rather than literal `<` and `>` characters.